element_deeplabcut/dlc.py

```diff
@@ -331,16 +281,17 @@ class Model(dj.Manual):
     task                 : varchar(32)  # task in the config yaml
     date                 : varchar(16)  # date in the config yaml
     iteration            : int          # iteration/version of this model
-    snapshot_idx         : int          # which snapshot for prediction (if -1, latest)
+    snapshotindex        : int          # which snapshot for prediction (if -1, latest)
     shuffle              : int          # which shuffle of the training dataset
-    training_fract_idx   : int          # which training set fraction to generate model
-    unique index (task, date, iteration, shuffle, snapshot_idx, training_fract_idx)
+    trainingsetindex     : int          # which training set fraction to generate model
+    unique index (task, date, iteration, shuffle, snapshotindex, trainingsetindex)
     scorer               : varchar(64)  # scorer/network name - DLC's GetScorerName()
     config_template      : longblob     # dictionary of the config for analyze_videos()

```

Comment:
In a previous version `config_template` conflicted with the foreign reference to another table with the same variable. I think, by this organization, we won't have the same conflict